plugins/servicenow/help.md

```diff
@@ -895,6 +895,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 7.1.1 - Create Incident: Resolved issue when nothing was passed to `additional_fields` input field 
```

Comment:
Will users be able to create without additional fields. If so, can you please update the message to reflect the same?